src/main/cli/cli.c

```diff
@@ -4814,7 +4814,7 @@ static void cliStatus(const char *cmdName, char *cmdline)
 #endif
 
 #ifdef BUILD_KEY
```

Comment:
```suggestion
  if(*build_key) {
```

Reply:
This change does not pass cli unit test

Reply:
IMO unittest shall be fixed (adding buildKey / releaseName stub or adding version.c to unittest)